TWLight/applications/templates/applications/application_list.html

```diff
@@ -1,229 +1,246 @@
+          {% comment %}Translator: On the page which shows a list of applications, coordinators must select which applications they wish to view. This string is found next to the selection buttons, and leads on to options such as "Under review", "Approved", or "Sent.{% endcomment %}
+          {% trans "View applications which are..." %}
+        </strong>
+        <div class="btn-group flex-wrap" role="nav" aria-labelledby="button_group">
+          <a href="{{ pending_url }}{% if filters %}?page=1{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}" type="button" class="btn btn-default {{ pending_class }}">
+            {% comment %}Translator: This describes the status of a group of applications.{% endcomment %}
+            {% trans "Pending review" %}
+          </a>
+          <a href="{{ approved_url }}{% if filters %}?page=1{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}" type="button" class="btn btn-default {{ approved_class }}">
+            {% comment %}Translator: This describes the status of a group of applications.{% endcomment %}
+            {% trans "Approved" %}
+          </a>
+          <a href="{{ rejected_url }}{% if filters %}?page=1{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}" type="button" class="btn btn-default {{ rejected_class }}">
+            {% comment %}Translator: This describes the status of a group of applications.{% endcomment %}
+            {% trans "Rejected" %}
+          </a>
+          <a href="{{ sent_url }}{% if filters %}?page=1{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}" type="button" class="btn btn-default {{ sent_class }}">
+            {% comment %}Translator: This describes the status of a group of applications.{% endcomment %}
+            {% trans "Sent" %}
+          </a>
+          <a href="{{ renewal_url }}{% if filters %}?page=1{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}" type="button" class="btn btn-default {{ renewal_class }}">
+            {% comment %}Translator: This describes the status of a group of applications.{% endcomment %}
+            {% trans "Up for renewal" %}
+          </a>
+        </div>
       </div>
     </div>
-  </div>
 
-  <h1>{{ title }}</h1>
+    <h1>{{ title }}</h1>
```

Comment:
Being a bit nitpicky here, but I think this header looks better with a `my-3` class

Reply:
Agreed, this looks much better!